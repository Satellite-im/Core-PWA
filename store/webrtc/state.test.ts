import InitialWebRTCState from '~/store/webrtc/state'

describe('init', () => {
  let inst: any

  beforeEach(() => {
    inst = InitialWebRTCState()
  })

  it('should return the initial settings state', () => {
    expect(inst).toMatchSnapshot()
  })

  it('should not return the initial settings state', () => {
    expect(inst).not.toEqual({})
  })
})
