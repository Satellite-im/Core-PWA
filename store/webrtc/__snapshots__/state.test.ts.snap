// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`init should return the initial settings state 1`] = `
Object {
  "activeCall": "",
  "activeStream": Object {
    "createdAt": 0,
  },
  "connectedPeer": "",
  "incomingCall": "",
  "initialized": false,
  "localTracks": Object {
    "audio": Object {},
    "video": Object {},
  },
  "remoteTracks": Object {
    "audio": Object {},
    "video": Object {},
  },
  "streaming": true,
}
`;
